packages/perseus-core/src/parse-perseus-json/regression-tests/parse-perseus-json-regression.test.ts

```diff
@@ -15,6 +15,9 @@ describe("parseAndTypecheckPerseusItem", () => {
 
         // This strange-looking assertion style results in the failure message
         // being printed if parsing fails, so the test is easier to debug.
+        expect((result as any).detail?.message).toBe(undefined);
```

Comment:
Does this work because if there is a message, then it will say something like "expected undefined but got <message>"? And the only time there is a message is when it fails?

Reply:
Yep!